lm_eval/models/huggingface.py

```diff
@@ -473,7 +479,7 @@ def _model_call(
         return self.model(**inputs, labels=labels["input_ids"])
 
     def _model_generate(
-        self, inputs: TokenSequence, max_tokens: int, stop: Optional[List[str]] = None
+        self, inputs: TokenSequence, max_tokens: int, min_tokens: int, stop: Optional[List[str]] = None
     ) -> Union[TokenSequence, List[str]]:
         stopping_criteria = stop_sequences_criteria(self.tokenizer, stop)
         generations = self.model.generate(
```

Comment:
Add `min_length` arg for Seq2Seq generation (I believe you can just set it to `min_tokens`)